### ISSUE TITLE
[Accessibility][Library] mobile menu icon not visible on mobile

### ISSUE THREAD

**Issue by bartoszglow** [adobe/aem-core-wcm-components#1760]
## Bug Report

**Current Behavior**
In the [Components Library](https://aemcomponents.dev) menu icon is not visible (below 1024px). It is possible to open in by clicking in the blank white space in the header however that is very unintuitive when there is no icon.

**Expected behavior/code**
Menu icon is visible on mobile.

**Environment**
- [Components Library](https://aemcomponents.dev) 
- Core Components 2.17.8

**Possible Solution**
---

**Additional context / Screenshots**
![IMG_1059](https://user-images.githubusercontent.com/5551340/129909008-6b00a271-cda9-4aa8-aa83-42c68ccf1fd0.jpeg)

**Comment by bartoszglow** [comment]
sync to Jira: @adobe export issue to Jira project SITES as Story

**Comment by bartoszglow** [comment]
@adobe export issue to Jira project SITES as Bug
